protocols/virtualdefund/virtualdefund.go

```diff
@@ -16,28 +17,56 @@ const (
 	WaitingForNothing                 protocols.WaitingFor = "WaitingForNothing"                 // Finished
 )
 
-// ObjectiveRequest represents a request to create a new virtual defunding objective.
-type ObjectiveRequest struct {
-	VirtualChannelId types.Destination
-}
-
 // Objective contains relevent information for the defund objective
 type Objective struct {
 	Status protocols.ObjectiveStatus
-	V      *channel.SingleHopVirtualChannel
+
+	// InitialOutcome is the initial outcome of the virtual channel
+	InitialOutcome outcome.SingleAssetExit
+
+	// PaidToBob is the amount that should be paid from Alice (participant 0) to Bob (participant 2)
+	PaidToBob *big.Int
+
+	// VFixed is the fixed channel information for the virtual channel
+	VFixed state.FixedPart
+
+	// Signatures are the signatures for the final virtual state from each participant
+	// Signatures are ordered by participant order: Signatures[0] is Alice's signature, Signatures[1] is Irene's signature, Signatures[2] is Bob's signature
+	// Signatures gets updated as participants sign and send states to each other.
+	Signatures [3]state.Signature
 
 	ToMyLeft  *consensus_channel.ConsensusChannel
 	ToMyRight *consensus_channel.ConsensusChannel
 
-	MyRole uint // index in the virtual funding protocol. 0 for Alice, 2 for Bob. Otherwise 1 for the intermediary.
-
+	// MyRole is the index of the participant in the participants list
+	// 0 is Alice
+	// 1 is Irene
+	// 2 is Bob
+	MyRole uint
 }
 
 const ObjectivePrefix = "VirtualDefund-"
 
+// finalState returns the final state for the virtual channel
+func (o Objective) finalState() state.State {
+	vp := state.VariablePart{Outcome: outcome.Exit{o.finalOutcome()}, TurnNum: 3, IsFinal: true}
```

Comment:
While `TurnNum: 3` is a reasonable final turn number, why not `2`?

Does it make sense to define a `FinalTurnNum` constant?

Reply:
Done in https://github.com/statechannels/go-nitro/pull/512/commits/a732771c053ddf5d1879765ff2ed2428032c5bc2